Automattic/wp-calypso
# Stats: Adding events for metered billing

Related to #84804

## Proposed Changes

* adding events for the new purchase page elements

## Testing Instructions



* apply `stats/plan-usage` and `stats/tier-upgrade-slider` feature flags
* in your network tag verify that pixel events fire after:
   * interacting with the slider on commercial and personal pages
   * clicking on the upgrade link from the usage bar
   * viewing the purchase page (with a proper `variant` parameter)

## Pre-merge Checklist
